lib/matplotlib/backends/backend_pdf.py

```diff
@@ -2700,6 +2700,9 @@ def print_pdf(self, filename, *,
             else:            # we opened the file above; now finish it off
                 file.close()
 
+    def draw(self, *args, **kwargs):
+        _no_op_draw(self.figure)
+        return super().draw(*args, **kwargs)
```

Comment:
I guess the superclass does nothing? It's a bit weird to do a no-op draw, and then go to the super's draw (which, from just looking at this one spot, could conceivably do _anything_.)

Reply:
In 99.9% of cases, the super-call will do nothing, but I do not want to assume that no one has done some (ill-advised?) multiple inheritance of our classes and was relying on hitting their draw.

That said, I think the name `_no_op_draw` is bad because it is doing an operation (updating our internal state), just not producing any output.